### ISSUE TITLE
Request: Steam workshop port

### ISSUE THREAD

**Issue by jdfr228** [staticvariablejames/ChooseYourOwnLump#4]
Would love to see this easily accessible in the Steam workshop, if possible.

**Comment by alainbryden** [comment]
For anyone else looking, it already has been ported here: https://steamcommunity.com/sharedfiles/filedetails/?id=3046293695 - but it's not official, so James may wish to leave this open.

**Comment by staticvariablejames** [comment]
Yes, that's an unofficial port, and it is of an older version of the mod.

I plan to port to Steam in the future, but it requires more work than just copy-pasting the mod code. Most prominently, Rigidel can never be active offline on Steam, so the mod and its documentation have to reflect that.
